hw2/01-coins.js

```diff
@@ -1,9 +1,83 @@
 /** Exercise 01 - Coins **/
```

Comment:
Looks great! Produces the expected output and is formatted correctly. It's modular and easy to follow, too. Good work!